feat: add lead

temirm opened the pull request:
Issue #3
Добавлены:
- разметка
- картинки
- стили для двух точек (414px и 1440px)

Стили для расширений между добавлю позже
(должна быть консистентность с блоком about)

barbylka committed:
Merge pull request #1 from barbylka/develop

fix: correct fonts path

temirm committed:
feat: add "lead"

Issue #3
Добавлены:
- разметка
- картинки
- стили для двух точек (414px и 1440px)

Стили для расширений между добавлю позже
(должна быть консистентность с блоком about)

barbylka commented on styles/index.css:
```diff
@@ -1,2 +1,15 @@
+@import url(../blocks/lead/__date/lead__date.css);
+@import url(../blocks/lead/__image/lead__image.css);
+@import url(../blocks/lead/__info/lead__info.css);
+@import url(../blocks/lead/__navigation-button/lead__navigation-button.css);
```
Видимо, старая строчка, если будет не актульна, можно удалить, а то браузер ругается

barbylka commented on index.html:
```diff
@@ -12,7 +12,35 @@
 
 		<main class="main">
-			<section class="lead"></section>
+			<section class="lead">				
```
Судя по табуляции, следует настроить Prettier, параметры настроек указаны в README, это позволит сохранить единство кодстайла в каждом файле, незаивисмо от того, кто его писал

temirm replied:
Вроде настроил Prettier. В будущем коммите табы должны быть из 2-х пробелов

barbylka reviewed (commented):
Все работает, все быстренько и четенько, здорово!

uladzimirfilipau commented:
Всё супер! Ещё хорошо будет добавить при наведении курсора на кнопку свойство cursor: pointer

temirm committed:
fix(lead): tabulation, pointer on button hover

Issue #3
Пофиксил:
- Табуляцию (2 пробела)
- Добавил cursor: pointer для кнопки
- Убрал лишнюю строку

temirm committed:
Merge branch 'develop' of github.com:barbylka/lubimovka into feature/3-add-lead

Теперь в проекте присутствуют две секции

temirm commented:
Ошибки исправил, из develop изменения смерджил. Локально обе секции вроде корректно отображаются. Могу вливать?
